apps/web/src/app/(signing)/sign/[token]/signing-field-container.tsx

```diff
@@ -182,6 +182,23 @@ export const SigningFieldContainer = ({
           </button>
         )}
 
+        {(field.type === FieldType.RADIO || field.type === FieldType.CHECKBOX) &&
+          field.fieldMeta?.label && (
+            <div
+              className={cn(
+                'absolute -top-16 left-0 right-0 rounded-md p-2 text-center text-xs text-gray-700',
```

Comment:
Can you consider extracting the label styling logic into a shared constant since it's duplicated in both `signing-field-container.tsx` and `field-item.tsx`?

Something like this will work.
```typescript
export const FIELD_LABEL_STYLES = {
  base: 'absolute -top-16 left-0 right-0 rounded-md p-2 text-center text-xs text-gray-700',
  unsigned: 'bg-foreground/5 border-border border',
  signed: 'bg-documenso-200 border-primary border'
};
```

Reply:
Sure, but is it worth it for 2 places only?

Reply:
currently, I dont think so but only if we'll extend it to ther places

Reply:
Makes sense.